aspnetcore/blazor/components/integration.md

```diff
@@ -186,9 +208,162 @@ Add a `Counter` component to the app with the interactive server render mode.
 }
 ```
 
-When the app is run, the `Counter` component is accessed at the `/counter` endpoint.
+When the app is run, the `Counter` component is accessed at `/counter`.
+
+### Enable interactive Auto and WebAssembly rendering
+
+Follow the guidance in the [Add static server Razor component rendering](#add-static-server-razor-component-rendering) section.
+
+Auto render mode components render on the server first, then render on the client after the Blazor bundle has been downloaded and the Blazor runtime activates. The interactive WebAssembly render mode renders only renders components on the client after the Blazor bundle is downloaded and the Blazor runtime activates. Keep in mind that component code is ***not*** private using the Auto or WebAssembly render modes. For more information, see <xref:blazor/components/render-modes>.
```

Comment:
Even with interactive WebAssembly rendering the components will by default still render first from the server as part of prerendering.

```suggestion
Components using the Auto render mode will initially use interactive server rendering, but then will switch to render on the client after the Blazor bundle has been downloaded and the Blazor runtime activates. Components using the WebAssembly render mode only render interactively on the client after the Blazor bundle is downloaded and the Blazor runtime activates. Keep in mind that when using the Auto or WebAssembly render modes, component code downloaded to the client is ***not*** private. For more information, see <xref:blazor/components/render-modes>.
```